test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,172 @@
+        b.set_input_text("#network-wireguard-settings-listen-port-input", str(m1_port))
+
+        # check ip addresses
+        b.set_input_text("#network-wireguard-settings-addresses-input", "10.0.0.1/24/56")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Addresses is not formatted correctly')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "10.0.0")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address 10.0.0')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "ten.one")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address ten.one')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "10 1")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address 10')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "1.2.3.4/")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid prefix or netmask')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "1.2.3.4  ,  5.6.7.8  1.2.3.4.5")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address 1.2.3.4.5')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", f"{m1_ip4}/24 1.2.3.4")
+
+        # peer
+        b.click("button:contains('Add peer')")
+        b.wait_visible("#network-wireguard-settings-peer-0")
+        b.set_input_text("#network-wireguard-settings-publickey-peer-0", m2_pubkey)
+        b.set_input_text("#network-wireguard-settings-endpoint-peer-0", f"192.168.100.12:{m2_port}")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-0", m2_ip4)
+        b.click("button:contains('Add peer')")
+        b.wait_visible("#network-wireguard-settings-peer-1")
+        b.set_input_text("#network-wireguard-settings-publickey-peer-1", m2_pubkey)
+        b.set_input_text("#network-wireguard-settings-endpoint-peer-1", f"192.168.100.12:{m2_port}")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-1", m2_ip4)
+        b.click("button#network-wireguard-settings-btn-close-peer-0")
+        b.wait_not_present("#network-wireguard-settings-peer-1")
+
+        b.click("button:contains('Save')")
+        b.wait_not_present("#network-wireguard-settings-dialog")
+        b.wait_in_text("#networking-interfaces th:contains('wg0') + td", f"1.2.3.4/32, {m1_ip4}/24")
+
+        m2.execute(f"wg set wg0 peer {m1_pubkey} allowed-ips {m1_ip4}/32")  # endpoint and port is not necessary for a peer if that peer estalishes the connectio first (i.e. the client)
+
+        # check connection over ipv4
+        try:
+            m1.execute(f"ping -c 5 {m2_ip4}")
+        except testlib.Error:
```

Comment:
`except (subprocess.CalledError, testlib.Error):` Sorry for the bad advice! This should fix error reporting for failures, see above.

Reply:
Assumed you meant `subprocess.CalledProcessError`.

Reply:
Err, yes :see_no_evil: